### PR TITLE
Compliance w/Export Filing Notice

### PR DESCRIPTION
**Pull request by cyberswat**
## The Problem:
According to legal we need to submit a notification to the BIS when appropriate.

## The Fix:
Add documentation to the product release checklist.

### CODE REVIEW

**Comment by cyberswat** [comment]
This addresses https://github.com/drud-admin/legal/issues/10

File: development/bis_template.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+From: Drud Technology, LLC  
+Tax ID:
```
Comment (rfay) [inline development/bis_template.md:2]: We want the tax id, right?

Comment (cyberswat) [reply development/bis_template.md:2]: We do, but I'm not sure I'm comfortable putting that into a public git repository.  I understand there is a process to retrieve the number from the IRS for companies not listed through the SEC, and that the information is generally "public".  However, I can't find it displayed in any of our public filings.

**Comment by beeradb** [comment]
Nudge. This looks a bit stale. Is there any real reviewing to happen here, or is it something we're required to have?

if it's the latter, I think we should just move forward with it. If it's the former, can we get some instructions on how to best review this?

**Review by rickmanelius — APPROVED** [review]
The only open item was whether or not we wanted to publicly show the Tax ID number. While it can be found through other means, I don't think it's necessary to display. I know I may sound weird about that, but I prefer to just insert and send versus have it in the community docs. We can always revisit, but the bulk of the work is done in this issue, so I say we merge and move on.

**Pull request merged**